src/binding/rdkafka/kafka_consumer_record.rs

```diff
@@ -26,8 +26,8 @@ impl ConsumerRecordDeserializer {
             // TODO create an error variant for invalid headers
             .ok_or(crate::message::Error::WrongEncoding {})?;
         for i in 0..headers.count() {
```

Comment:
Not sure if that applies here, but IIRC is saw a new `.iter()` function which allows one to iterate over headers.